src/import/YAMLschema.ts

```diff
@@ -0,0 +1,449 @@
+        reference: { type: 'string' },
+        type: { type: 'string' },
+        identifier: { $ref: '#/definitions/identifier' },
+        display: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    identifier: {
+      type: 'object',
+      properties: {
+        use: { type: 'string' },
+        type: {
+          oneOf: [{ $ref: '#/definitions/codeableConcept' }, { type: 'string' }]
+        },
+        system: { type: 'string' },
+        value: { type: 'string' },
+        period: { $ref: '#/definitions/period' },
+        assigner: { $ref: '#/definitions/configReference' }
+      },
+      additionalProperties: false
+    },
+    jurisdiction: {
+      oneOf: [{ type: 'string' }, { $ref: '#/definitions/codeableConcept' }]
+    },
+    dependencyDetails: {
+      type: 'object',
+      properties: {
+        id: { type: 'string' },
+        uri: { type: 'string' },
+        version: { type: ['string', 'number'] },
+        reason: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    configResource: {
+      type: 'object',
+      properties: {
+        reference: {
+          type: 'object',
+          properties: {
+            reference: { type: 'string' },
+            type: { type: 'string' },
+            identifier: { $ref: '#/definitions/identifier' },
+            display: { type: 'string' }
+          },
+          additionalProperties: false
+        },
+        fhirVersion: {
+          oneOf: [{ type: 'string' }, { type: 'array', items: { type: 'string' } }]
+        },
+        name: { type: 'string' },
+        description: { type: 'string' },
+        exampleBoolean: { type: 'boolean' },
+        exampleCanonical: { type: 'string' },
+        isExample: { type: 'boolean' },
+        profile: {
+          oneOf: [{ type: 'string' }, { type: 'array', items: { type: 'string' } }]
+        },
+        groupingId: { type: 'string' }
+      },
+      additionalProperties: false
+    },
+    configPage: {
+      oneOf: [
+        {
+          type: 'null'
+        },
+        {
+          type: 'object',
+          properties: {
+            title: { type: 'string' },
+            generation: { type: 'string' },
+            extension: { type: 'array', items: { $ref: '#/definitions/extension' } },
+            modifierExtension: { type: 'array', items: { $ref: '#/definitions/extension' } },
+            name: { type: 'string' },
+            sourceUrl: { type: 'string' },
+            sourceString: { type: 'string' },
+            sourceMarkdown: { type: 'string' }
+          },
+          patternProperties: {
+            '\\.(md|xml|html)$': {
+              $ref: '#/definitions/configPage'
+            }
+          },
+          additionalProperties: {
+            oneOf: [
+              // { $ref: '#/definitions/configPage' },
+              { type: 'string' },
+              { type: 'array', items: { $ref: '#/definitions/extension' } }
+            ]
+          }
```

Comment:
What are these `additionalProperties` intended to cover?

Reply:
I put that in to align with `[key: string]: YAMLConfigurationPage | string | Extension[];` in the `YAMLConfigurationPage` type. But, looking at it more closely, the purpose it serves in the type definition is to allow for nested pages, which is covered by the `patternProperties` in the schema, so these `additionalProperties` can be removed.

Reply:
Also, this points out that the message when a page doesn't have the extension we like, the current implementation just says "unrecognized property". In this case, I think it's worth it for me to try to make the message more specific.